content/02.introduction.md

```diff
@@ -14,9 +14,9 @@ At this time, over 11,000 deaths had been reported worldwide.
 [Note: Maybe add a graph here, update as new reports come out.]
 
 COVID-19 is caused by the newly identified severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2).
-SARS-CoV-2 is a coronavirus, a type of RNA virus known to cause respitatory infections in humans and other species.
-Other well-known coronaviruses include those associated with previous infectious diseases of global concern, including Severe Accute Repiratory Syndrome (SARS) and Middle East respiratory syndrome (MERS); however, some coronaviruses are associated with less virulent illnesses including the common cold.
-The SARS-CoV-2 virus was unknown until approximately January 12, 2020, when Chinese officials released its genetic sequence to aid in worldwide efforts to diagnose the disease [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200121-sitrep-1-2019-ncov.pdf].
+SARS-CoV-2 is a coronavirus, a family of RNA viruses known to cause respiratory and intestinal infections in humans and other species.
+Infectious diseases of global concern have previously been associated with coronaviruses, including Severe Acute Respiratory Syndrome (SARS-CoV) and Middle East respiratory syndrome (MERS-CoV); however, neither of these reached pandemic status, owing to proper containment procedures (SARS) or intrinsic limitations in virus transmission (MERS). Additionally, there are four endemic human coronaviruses that rarely progress beyond the mild symptoms associated with the common cold.
```

Comment:
```suggestion
Infectious diseases of global concern have previously been associated with coronaviruses, including Severe Acute Respiratory Syndrome (SARS-CoV) and Middle East respiratory syndrome (MERS-CoV); however, neither of these reached pandemic status, owing to proper containment procedures (SARS) or intrinsic limitations in virus transmission (MERS).
Additionally, there are four endemic human coronaviruses that rarely progress beyond the mild symptoms associated with the common cold.
```
One sentence per line for easier commenting.
Also, is there a good review on the four endemic human coronaviruses? and/or SARS/MERS? Manubot has a robust citation infrastructure by DOI, PMID, etc, so it'd be good to start getting some examples of it in the paper :)

Reply:
Sorry, still getting used to github! 
Here are the two reviews I've been using to brush up on coronaviruses:
https://www.nature.com/articles/s41579-018-0118-9
and
https://www.nature.com/articles/nrmicro.2016.81

Reply:
Ok! I tried to cite those (both for the first part, just the one for the second one which was how it looked like the content was distributed to me). If you select "commit suggestion" it'll add these refs and hopefully build successfully 🤞 

The manubot citation approach is described here in case it's helpful:
https://github.com/greenelab/covid19-review/blob/master/USAGE.md#citations

Reply:
Hi @rmrussell !
FYI, to add citations, you can cite using the format Casey used above:
```
Citation by DOI [@doi:10.7554/eLife.32822].
Citation by PubMed Central ID [@pmcid:PMC6103790]
```
Full info available [here](https://github.com/greenelab/covid19-review/blob/master/content/80.example-formatting.md#Citations)
They will automatically generate in [this preview](https://greenelab.github.io/covid19-review/) once it has a little more content :)

Thanks so much for working on this! It's already sounding way better!

Reply:
I just tried to make sure I got the syntax right myself 😂  Hope this builds successfully!